contrib/Shopify/sarama/sarama_test.go

```diff
@@ -262,6 +266,7 @@ func TestAsyncProducer(t *testing.T) {
 
 func newMockBroker(t *testing.T) *sarama.MockBroker {
 	broker := sarama.NewMockBroker(t, 1)
+	fmt.Println(broker.Addr())
```

Comment:
maybe `t.Logf("broker address: %v, broker.Addr())` if we really want it in test output

Reply:
no, missed it.